### DISCUSSION TITLE
Change babel,es version (or other?) settings

### DISCUSSION THREAD

**Discussion by idc77** [quasarframework/quasar#16915] [General - Components / Directives / etc] [answered]
See this vuex-orm issue:
https://github.com/vuex-orm/vuex-orm/issues/220

Something like
```js
import { Model } from '@vuex-orm/core'

export default class User extends Model {
    static entity = 'users'

    static fields () {
        return {}
    }
}
```

This `static entity = 'users'` created an error `unexpected token =`.

Now, in reponse https://github.com/vuex-orm/vuex-orm/issues/220#issuecomment-420115464
he has a few way to solve this.

As far as I understand, this is some setting that says which ECMAscript version is supported.

How can I make the above code valid, without needing to rewrite
```
static entity = 'users'
```

into

```
static get entity() {
  return 'users'
}
```
?

**Comment by rstoenescu** [comment, marked as answer]
Hi,

If you are using q/app-webpack then you already have a `/babel.config.cjs` file there in your project folder.
With q/app-vite you may need the [Vite plugin legacy](https://github.com/vitejs/vite/tree/main/packages/plugin-legacy). Install it then add it to your `/quasar.config.js` file > build > vitePlugins [Adding Vite plugins](https://quasar.dev/quasar-cli-vite/handling-vite#adding-vite-plugins).
